src/main/java/frc/robot/constants/KitBot2022Constants.java

```diff
@@ -65,27 +65,27 @@ public boolean driveMotorRightFollowerInverted() {
     }
 
     @Override
-    public boolean hasClimber2020() {
-        return false;
+    public boolean hasClimber2022() {
+        return true;
     }
 
     @Override
-    public int climber2020MotorId() {
-        return 0;
+    public int climber2022MotorId() {
+        return 6;
```

Comment:
this id is for robot2022, not kitbot2022
```suggestion
        return 0;
```